src/main/java/org/sonarlint/intellij/issue/IssueStore.java

```diff
@@ -26,13 +26,13 @@
 import com.intellij.psi.PsiElement;
 import com.intellij.psi.PsiFile;
 import com.intellij.util.messages.MessageBusConnection;
+import org.sonarlint.intellij.issue.tracking.Input;
+import org.sonarlint.intellij.issue.tracking.Tracker;
+import org.sonarlint.intellij.issue.tracking.Tracking;
 import org.sonarlint.intellij.messages.AnalysisResultsListener;
 
 import javax.annotation.concurrent.ThreadSafe;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.Map;
+import java.util.*;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Explicitly import the specific classes needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2208)